src/AutoComplete/StyledAutoCompleteMultiValue.js

```diff
@@ -10,9 +10,13 @@ import Icon from '../Icon';
 const StyledAutoCompleteMultiValue = ({
   data,
   data: { label },
+  paletteColor,
   removeProps
 }) => {
-  const paletteColor = colorForString(label, Object.keys(palette));
+  const color =
+    paletteColor === null
```

Comment:
awkward because `''` is the current way to get the neutral palette. https://www.pivotaltracker.com/story/show/172611876 is related